CI/bitrise.yml

```diff
@@ -38,7 +38,7 @@ workflows:
 
             set -e
 
-            sh bin/swift5-all.sh
+            sh bin/generate-samples.sh bin/configs/swift5-*.yaml
```

Comment:
Aren't these samples also run in Drone when we invoke `/bin/bash bin/generate-samples.sh` why run them twice?

Reply:
Different CI run in parallel and don't have a way to know this has been run.

See my comment in this PR about potentially using GitHub Workflow

Reply:
Right. In other words, the jobs run independently on different CI providers.

@spacether if you've further questions on the CI, PM me via Slack as I setup most of them.

Reply:
Thank you. Yup I was aware that they run in parallel independently. What I didn't understand was why we need each of them. Jim pointed me to https://github.com/OpenAPITools/openapi-generator/wiki/Integration-Tests
Which shows that we use bitrise to test swift 4/5 so it looks like the reason that we generate them in both locations is that:
- in drone we are testing JDK11 test, OCaml client, etc and need the samples generated for that testing
- in bitrise we are specifically testing switft4/5 so we need to run sample generation prior to that